src/pages/sok.tsx

```diff
@@ -0,0 +1,213 @@
+          <u>{t.sok.extra}</u>
+        </button>
+
+        {/*MODAL*/}
+        {modalState && (
+          <div
+            className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-60 backdrop-blur-lg"
+            ref={modalRef}
+            onClick={handleOverlayClick}
+          >
+            <div
+              className={`bg-white bg-opacity-70 w-[500px] max-h-[80vh] overflow-y-auto pb-5 flex flex-col rounded-3xl`}
+              style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
+            >
+              <div className="relative py-[0px] justify-center flex flex-row">
+                <img src={image} />
+
+                <button
+                  className="absolute top-5 right-3 w-[50px] h-[50px] flex items-center justify-center"
+                  onClick={closeModal}
+                >
+                  <div className="absolute h-[50px] w-[5px] bg-white rounded-md rotate-45"></div>
+                  <div className="absolute h-[50px] w-[5px] bg-white rounded-md -rotate-45"></div>
+                </button>
+              </div>
+              <div className="px-5 mt-5">
+                <h2 className="text-center text-3xl text-black">
+                  {eventInfo[0]}
+                </h2>
+                <h3 className="text-center text-cerise text-2xl mt-2"></h3>
+                <ul className="list-none mt-2 text-center">
+                  {roles.map((role, index) => (
+                    <li key={index} className={index !== 0 ? "mt-8" : ""}>
+                      {role.split("\n").map((line, i) => (
+                        <p
+                          key={i}
+                          className={
+                            i === 0 ? "ml-0 text-left text-cerise" : "ml-0"
+                          }
+                        >
+                          {line}
+                        </p>
+                      ))}
+                    </li>
+                  ))}
+                </ul>
+                <p className="text-black text-start mt-5">
+                  {/* Content here */}
+                </p>
+              </div>
+            </div>
+          </div>
+        )}
+      </div>
+    </div>
+  );
+}
+
+export default function Students() {
+  const t = useLocale();
+
+  return (
+    <div className="pt-[200px] pb-[300px]">
+      <h1 className="text-5xl text-cerise font-medium text-center uppercase">
+        {" "}
+        {t.sok.header}
+      </h1>
+      <div className="flex flex-row items-center justify-center mt-[50px]">
+        <p className="text-white w-[300px] sm:w-[600px] text-center">
+          {t.sok.description}
+        </p>
+      </div>
+      <a href="https://forms.gle/VyigeGiQuEXgE9eS8" target="_blank">
+      <div className="flex flex-row items-center justify-center">
+        <button className="w-[200px] bg-cerise rounded-full h-[60px] mt-[50px]">
+          <p className="text-white text-center"> {t.sok.search}</p>
+        </button>
```

Comment:
Text padding could be less in button IMO but its a preference
![image](https://github.com/datasektionen/ddagen/assets/66337956/a910f87e-08e8-4bd3-9a96-d27f54d02839)

Reply:
I made button smaller overall, which gives a similar effect i believe